grammar.js

```diff
@@ -787,38 +801,14 @@ module.exports = grammar({
 
     update: $ => seq(
       $.keyword_update,
-      choice(
-        $._single_table_update,
-        $._multi_table_update,
-      ),
-    ),
-
-    _single_table_update: $ => seq(
-      optional(
-        $.keyword_only,
-      ),
-      $.table_reference,
-      $.keyword_set,
-      comma_list($.assignment, true),
-      optional($.where),
-      optional($.order_by),
-      optional($.limit),
-    ),
-
-    _multi_table_update: $ => seq(
-      $._table_references,
+      optional($.keyword_only),
+      comma_list($.relation, true),
+      repeat($.join),
```

Comment:
Postgres' update-join syntax is different, with the `set` clause coming before a more standard `from`-`join` construction that specifies the additional relations (the affected table is still specified right after the initial `update`). I think this is a net zero change for PG but I'll leave an example update statement with the tests if you want to try fitting that in too.

Reply:
done!